public/app/core/services/ng_react.ts

```diff
@@ -54,7 +54,7 @@ function applied(fn: any, scope: any) {
   if (fn.wrappedInApply) {
     return fn;
   }
-  //tslint:disable-next-line:only-arrow-functions
+  // this had the equivalent of `eslint-disable-next-line prefer-arrow/prefer-arrow-functions`
```

Comment:
Not sure what you mean by this comment?

Reply:
Some tslint rules are slightly different in eslint (and typescript-eslint), which would've resulted in many source code file changes. They'll be handled soon in another PR, but I wanted to keep a record of the previous state without retaining any tslint comments.

Reply:
gottcha